Classes/Emogrifier.php

```diff
@@ -1077,6 +1078,14 @@ function ($matches) use (&$media) {
             $cssWithoutComments
         );
 
+		//Keep @imports
+		$imports = '';
+		preg_match_all('/^\\s*@import\\s[^;]+;/misU', $css, $importsMatches, PREG_PATTERN_ORDER);
+		if( !empty($importsMatches[0]) ){
+			$importsArray = reset($importsMatches);
```

Comment:
reset was giving me the first element of the array, same as $importsMatches[0] so I did remove the reset and replaced with $importsMatches[0]. Can you give me some more info what  is not PSR-2. Thanks in advanced man.